servicedirectory/Gemfile.lock

```diff
@@ -0,0 +1,89 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    addressable (2.7.0)
+      public_suffix (>= 2.0.2, < 5.0)
+    diff-lcs (1.3)
+    faraday (1.0.1)
+      multipart-post (>= 1.2, < 3)
+    gapic-common (0.2.0)
+      google-protobuf (~> 3.2)
+      googleapis-common-protos (>= 1.3.9, < 2.0)
+      googleapis-common-protos-types (>= 1.0.4, < 2.0)
+      googleauth (~> 0.9)
+      grpc (~> 1.25)
+    google-cloud-core (1.5.0)
+      google-cloud-env (~> 1.0)
+      google-cloud-errors (~> 1.0)
+    google-cloud-env (1.3.1)
+      faraday (>= 0.17.3, < 2.0)
+    google-cloud-errors (1.0.0)
+    google-cloud-service_directory (0.2.0)
+      google-cloud-core (~> 1.5)
+      google-cloud-service_directory-v1beta1 (~> 0.1)
+    google-cloud-service_directory-v1beta1 (0.2.1)
+      gapic-common (~> 0.2)
+      google-cloud-errors (~> 1.0)
+      grpc-google-iam-v1 (~> 0.6.9)
+    google-protobuf (3.11.4)
+    googleapis-common-protos (1.3.10)
+      google-protobuf (~> 3.11)
+      googleapis-common-protos-types (>= 1.0.5, < 2.0)
+      grpc (~> 1.27)
+    googleapis-common-protos-types (1.0.5)
+      google-protobuf (~> 3.11)
+    googleauth (0.11.0)
+      faraday (>= 0.17.3, < 2.0)
+      jwt (>= 1.4, < 3.0)
+      memoist (~> 0.16)
+      multi_json (~> 1.11)
+      os (>= 0.9, < 2.0)
+      signet (~> 0.12)
+    grpc (1.28.0)
+      google-protobuf (~> 3.11)
+      googleapis-common-protos-types (~> 1.0)
+    grpc-google-iam-v1 (0.6.10)
+      google-protobuf (~> 3.11)
+      googleapis-common-protos (>= 1.3.10, < 2.0)
+      grpc (~> 1.27)
+    jwt (2.2.1)
+    memoist (0.16.2)
+    multi_json (1.14.1)
+    multipart-post (2.1.1)
+    os (1.1.0)
+    public_suffix (4.0.4)
+    rspec (3.9.0)
+      rspec-core (~> 3.9.0)
+      rspec-expectations (~> 3.9.0)
+      rspec-mocks (~> 3.9.0)
+    rspec-core (3.9.1)
+      rspec-support (~> 3.9.1)
+    rspec-expectations (3.9.1)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.9.0)
+    rspec-mocks (3.9.1)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.9.0)
+    rspec-retry (0.6.2)
+      rspec-core (> 3.3)
+    rspec-support (3.9.2)
+    rspec_junit_formatter (0.4.1)
+      rspec-core (>= 2, < 4, != 2.12.0)
+    signet (0.14.0)
+      addressable (~> 2.3)
+      faraday (>= 0.17.3, < 2.0)
+      jwt (>= 1.5, < 3.0)
+      multi_json (~> 1.10)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  google-cloud-service_directory (~> 0.2.0)
+  google-cloud-service_directory-v1beta1 (~> 0.2.1)
+  rspec
+  rspec-retry
+  rspec_junit_formatter
+
+BUNDLED WITH
+   2.1.4
```

Comment:
ci might be grumpy with this. Try 1.17.3

Reply:
Done.